setup.cfg

```diff
@@ -32,6 +32,7 @@ console_scripts =
    pbench-list-triggers = pbench.cli.agent.commands.triggers.list:main
    pbench-register-tool-trigger = pbench.cli.agent.commands.triggers.register:main
    pbench-server = pbench.cli.server.shell:main
+   pbench-user-create = pbench.cli.server.user_create:main
```

Comment:
We also have to modify the `server/Makefile` to copy the new `pbench-user-create` command.

Reply:
We should probably follow the lead of the `agent/Makefile` for this -- it uses a list of tools to copy, so adding new tools just means adding an entry to the list.  (Currently, I think there is only one tool (`pbench-config`) so there hasn't been a need for a "list", but now that there are two....)

Reply:
Thanks, @webbnh; I had thought to make a similar comment after comparing the two Makefiles, but forgot to come back.

Reply:
> Currently, I think there is only one tool (`pbench-config`) so there hasn't been a need for a "list", but now that there are two....

My bad:  `pbench-config` has not yet been Clickinated.  But, under the assumption that there will indeed be future Click-based scripts as part of the server installation, let's keep the list.